swig/std_set.i

```diff
@@ -0,0 +1,124 @@
+/*=========================================================================
```

Comment:
Is this file only available in some newer version of SWIG and that's why we have to include it in our distribution? I can see that other types such as std_map.i are not included, we just reference them.

Reply:
Yes, don't know why but the C# template for std:set is not provided.